post-scan-actions/azure-python-promote-or-quarantine/README.md

```diff
@@ -0,0 +1,74 @@
+    - Create a 'Quarantine storage account' to receive quarantined files. Example: `fssquarantine`.
+    - In the Azure portal, go to **Storage account** > Quarantine storage account > **Access keys**, click **Show keys**, and copy the value of **Connection string** of either key.
+1. **Find the scanning blob storage account resource ID and the service bus topic resource ID**
+    - In the Azure portal, go to **Resource group** > your storage stack > **Deployments > storageStack > Outputs**.
+    - Copy the value of **blobStorageAccountResourceID** to a temporary location. Example: `/subscriptions/12345678-1111-1111-1111-123456789012/resourceGroups/my-storages/providers/Microsoft.Storage/storageAccounts/scanningstorage`.
+    - Copy the value of **scanResultTopicResourceID** to a temporary location. Example: `/subscriptions/12345678-1111-1111-1111-123456789012/resourceGroups/my-storage-stack/providers/Microsoft.ServiceBus/namespaces/tmsrt0abcd56789abcd/topics/scan-result-topic`.
+
+## Installation
+
+### Deploy Azure function app
+
+1. Login Azure CLI and switch to the Azure subscription to deploy resources:
+
+    ```bash
+    az login
+    az account set -s $AZURE_SUBSCRIPTION_NAME
+    ```
+
+1. Prepare a resource group for deploying the resources. To create a new resource group, execute the following command:
+
+    ```bash
+    az group create -n $RESOURCE_GROUP_NAME -l $AZURE_REGION
+    ```
+
+1. Deploy the function app and required resources:
+
+    ```bash
+    az deployment group create \
+        --name fss-promote-or-quarantine-plugin-deployment \
+        --resource-group $RESOURCE_GROUP_NAME \
+        --template-uri https://raw.githubusercontent.com/trendmicro/cloudone-filestorage-plugins/master/post-scan-actions/azure-python-promote-or-quarantine/template.json \
+        -p promoteOrQuarantineFunctionName=$FUNCTION_NAME \
+        scanningStorageAccountResourceID=$BLOB_STORAGE_ACCOUNT_RESOURCE_ID \
+        scanResultTopicResourceID=$SCAN_RESULT_TOPIC_RESOURCE_ID \
+        promoteStorageAccountConnectionString="$PROMOTE_STORAGE_ACCOUNT_CONNECTION_STRING" \
+        promoteMode=move \
+        quarantineStorageAccountConnectionString="$QUARANTINE_STORAGE_ACCOUNT_CONNECTION_STRING" \
+        quarantineMode=move
+    ```
+
+    To copy the scanned files from scanning storage account to the promote or quarantine storage account instead of moving them, modify the value of `promoteMode` or `quarantineMode` parameter to `copy`:
+
+    ``` bash
+    promoteMode=copy
+    quarantineMode=copy
+    ```
+
+### Public Azure function
+
+1. Download [cloudone-filestorage-plugins repository](https://github.com/trendmicro/cloudone-filestorage-plugins/tree/feat/azure_quarantine_promotion_plugin)
+1. Navigate to `post-scan-actions/azure-python-promote-or-quarantine/promoteOrQuarantineFunction`
+1. Deploy the function to the function app:
+
+    ```bash
+    func azure functionapp publish $FUNCTION_NAME
```

Comment:
Ｈow to set `quarantineStorageAccountConnectionString` or `QUARANTINE_STORAGE_ACCOUNT_CONNECTION_STRING` in this deployment method?